EIPS/eip-1.md

```diff
@@ -60,9 +60,11 @@ The best way to get client implementers to review your EIP is to present it on a
 
-These calls generally result in a "rough consensus" around what EIPs should be implemented. This "rough consensus" rests on the assumptions that EIPs are not contentious enough to cause a network split and that they are technically sound.
+These calls generally result in a "rough consensus" around what EIPs should be implemented. This "rough consensus" rests on the assumptions that EIPs are technically sound and their implementation is consistent with the best knowledge available of the political will of the Ethereum community.
 
-:warning: The EIPs process and AllCoreDevs call were not designed to address contentious non-technical issues, but, due to the lack of other ways to address these, often end up entangled in them. This puts the burden on client implementers to try and gauge community sentiment, which hinders the technical coordination function of EIPs and AllCoreDevs calls. If you are shepherding an EIP, you can make the process of building community consensus easier by making sure that [the Ethereum Magicians forum] thread for your EIP includes or links to as much of the community discussion as possible and that various stakeholders are well-represented.
+:warning: The EIPs process is not intended to fully capture the political will of the community. As a technical standardization process, the best case scenario for EIPs that may be politically contentious is to standardize the technical details of a proposal as a reference point for political debate to occur. No political decisions occur as a result of the EIP process, as political speech cannot itself be subject to standardization.
+
+:warning: The client implementers will do their best to be as aware of the political climate surrounding a proposal as they can find the time and energy to do so, but their failure to understand all political discussion around contentious issues should not be considered a shortcoming of the process, nor a value judgement of their effectiveness at this role. As the shepherd of an EIP, you can make the process of building community consensus easier by making sure that [the Ethereum Magicians forum] thread for your EIP includes or links to as much of the community discussion as possible and that various stakeholders are well-represented. This collective sentiment of the community should be presented, in full, with detailed descriptions of the various facts and opinions the community holds surrounding the proposal, to the client implementors on the AllCoreDevs call, so they have the most relevant information available to make decisions of what to implement in their clients on behalf of their users. The community ultimately uses the software that client developers implement to signal the configuration of the main Ethereum network during critical upgrade periods, and the decisions the client developers make should reflect the simplest configuration possible for what they expect the communities' decisions to be.
```

Comment:
> [the Ethereum Magicians forum]

Link missing to https://ethereum-magicians.org/

Reply:
```suggestion
:warning: The client implementers will do their best to be as aware of the political climate surrounding a proposal as they can find the time and energy to do so, but their failure to understand all political discussion around contentious issues should not be considered a shortcoming of the process, nor a value judgement of their effectiveness at this role. As the shepherd of an EIP, you can make the process of building community consensus easier by making sure that [the Ethereum Magicians forum](https://ethereum-magicians.org/) thread for your EIP includes or links to as much of the community discussion as possible and that various stakeholders are well-represented. This collective sentiment of the community should be presented, in full, with detailed descriptions of the various facts and opinions the community holds surrounding the proposal, to the client implementors on the AllCoreDevs call, so they have the most relevant information available to make decisions of what to implement in their clients on behalf of their users. The community ultimately uses the software that client developers implement to signal the configuration of the main Ethereum network during critical upgrade periods, and the decisions the client developers make should reflect the simplest configuration possible for what they expect the communities' decisions to be.
```